README.md

```diff
@@ -184,6 +185,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [Tomba.io](https://tomba.io/) - With 430+ million email addresses indexed, effective search filters, and deliverability checks, Tomba's email finder is its most powerful tool
 - [Snov Email Finder](https://snov.io/email-finder) Find any email. Anywhere
 - [EmailSearch.io](https://emailsearch.io/) - Find any emails and phones from a domain, Linkedin, name, and company
+- [Mailboxvalidator](https://www.mailboxvalidator.com/) - Email verification service to validate email addresses for accuracy and deliverability
```

Comment:
The description should be `Secure and reliable email validation service to check for invalid email addresses`